pkg/kritis/crd/securitypolicy/violation.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package securitypolicy
+
+import (
+	"fmt"
+	"github.com/grafeas/kritis/pkg/kritis/apis/kritis/v1beta1"
+	"github.com/grafeas/kritis/pkg/kritis/constants"
+	"github.com/grafeas/kritis/pkg/kritis/metadata"
+)
+
+type violation string
```

Comment:
Ah, here I actually meant to make the UnqualifiedImageViolationReason function return a "violation" string, instead of using this type as the enum.

Reply:
Oh, thanks for clarifying. I'll return the "violation" string and switch the enum back to ints, is that correct?

Reply:
right, thanks! One last nit: since these are Public methods, you should make the Violation type Public as well.